pkg/cmd/server/start/start_master.go

```diff
@@ -378,7 +380,7 @@ func (m *Master) Start() error {
 	}
 
 	if m.api {
-		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
+		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
I agree, it's shaky.

I could get on board with logging message IDs, but that would still leave us scraping logs for message IDs (which we hope are unique) and not having any good way to check that the message source actually emits that message ID (someone could look at it, wonder why it was there, remove/change it, and not suffer any failed tests). So I'm not sure it solves the problem any better, unless we externalize the message ID to enable the same kind of testing demonstrated in this PR.

We also may have no choice but scraping human-directed messages in some situations where they come from somewhere outside our influence and/or can't be changed.

You could argue that OpenShift shouldn't force us to look through human-directed server logs to figure out what is going on :)

Any ideas on where to go with this?